plugins/aea-crypto-fetchai/setup.py

```diff
@@ -42,4 +42,23 @@
         "aea.ledger_apis": ["fetchai = aea_crypto_fetchai:FetchAIApi"],
         "aea.faucet_apis": ["fetchai = aea_crypto_fetchai:FetchAIFaucetApi"],
     },
+    classifiers=[
+        "Environment :: Console",
+        "Environment :: Web Environment",
+        "Development Status :: 2 - Pre-Alpha",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Natural Language :: English",
+        "Operating System :: MacOS",
+        "Operating System :: Microsoft",
+        "Operating System :: Unix",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Topic :: Communications",
+        "Topic :: Internet",
+        "Topic :: Software Development",
+        "Framework :: AEA",
```

Comment:
`Framework :: AEA` was kind of missing. It is also used by Pytest:
https://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others

Reply:
the others are copy-pasted from main `setup.py`, I have only removed some `Topic`s (Scientific/Engineering, System)